x-pack/plugins/cross_cluster_replication/public/app/sections/home/follower_indices_list/components/context_menu/context_menu.js

```diff
@@ -73,7 +73,7 @@ export class ContextMenuUi extends PureComponent {
 
     const button = (
       <EuiButton
-        data-test-subj="followerIndexContextMenuButton"
+        data-test-subj="ccrFollowerIndexContextMenuButton"
```

Comment:
Because there may be multiple instances of this visible at the same time (above the table and in the flyout), maybe this should be assigned by the consumer, so they'll be unique? We can use the same `...rest` trick I suggested for supporting pass-through props, above.

Reply:
since the context menu is a popover, and EUI popovers are singletons, we won't ever run into duplication of these test subjects when the flyout is open

Reply:
Ah I see, I should clarify that I'm referring to the button itself, not the context menu. I did a quick test and I was able to select two instances of this button by checking a row and opening the flyout:

![image](https://user-images.githubusercontent.com/1238659/52829227-c294f580-3080-11e9-8c73-a0a933399c69.png)

Reply:
oh! I see 😄